cores/esp8266/Esp.cpp

```diff
@@ -177,6 +178,12 @@ uint16_t EspClass::getMaxFreeBlockSize(void)
     return umm_max_block_size();
 }
 
+uint32_t EspClass::getFreeContStack()
+{
+    extern cont_t* g_pcont;
```

Comment:
This declaration is already done in `cont.h`

As a general question, why don't we write these trivial functions directly inside the class ?
This would have the effect of reducing the final image size if they are not used.

Reply:
Damn right !
I was totally unaware of that and I thought that was the LTO feature that we don't have.
So I uselessly separated some code from Esp.cpp namely Esp-frag.cpp and Esp-version.cpp.
Thanks for opening my eyes, I will seek details on the differences between these two gcc/ld features.